src/app/app.component.ts

```diff
@@ -126,8 +126,57 @@ export class MyApp implements OnInit, AfterViewInit {
       this.handleBackButton();
       this.appRatingService.checkInitialDate();
       this.getUtmParameter();
+      this.checkForCodeUpdates();
+      this.platform.resume.subscribe(() => this.checkForCodeUpdates());
```

Comment:
We are already subscribing for platform resume in line no 276. we can move the code to that block.

Reply:
is it required to check for updates always when the app resumes?

Reply:
Yes Swayanjgit we may need to check for every update